service/test/integration2_test.go

```diff
@@ -1581,6 +1581,55 @@ func TestClientServerFunctionCall(t *testing.T) {
 	})
 }
 
+func TestClientServerFunctionCallEmbeddedFiled(t *testing.T) {
```

Comment:
Instead of writing this you should add a test case to TestCallFunction in service/test/variables_test.go

It should look like this:

```
{`d.Base.Method()`, []string{ `:int:4` }, nil },
{`d.Method()`, []string{ `:int:4` }, nil },
```